roles/dns/manage-dns-zones/tasks/named/prereq.yml

```diff
@@ -0,0 +1,10 @@
+---
+
```

Comment:
Here maybe use the ansible tempdir module?

Reply:
Yes, good catch. this was carry-over code from the old role (which was created before Ansible had the tempdir module). I'll submit an update.

Reply:
Done